source/cpptooling/data/representation.d

```diff
@@ -1381,14 +1382,6 @@ const:
     }
 }
 
-/// Dictates how the namespaces are merged.
-enum MergeMode {
-    /// Merge everything except nested namespaces.
-    shallow,
-    /// Merge everything.
-    full
-}
-
 @safe struct CppNamespace {
```

Comment:
Did not run any tests unfortunately, buuuuuuut I found the problem. I believe that we are using put(CppNamespace) in CppRoot wrongly. 

In our process() function we are creating a new CppRoot to put all our namespaces that we filtered out (line 98 in backend/fuzzvariant.d), either something broke in the put function or we should be doing something else.